<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkClientView</class>
 <widget class="QWidget" name="QmitkClientView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="uriLineEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="getSinglePushButton">
     <property name="text">
      <string>GET (single tasks)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="getMultiplePushButton">
     <property name="text">
      <string>GET (multiple tasks)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="putPushButton">
     <property name="text">
      <string>PUT</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="postPushButton">
     <property name="text">
      <string>POST</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="responseLabel">
     <property name="text">
      <string>This label changes after request</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
